src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const everyDayRent = 40;
+  let totalSum = everyDayRent * days;
+
+  if (days >= 7) {
+    totalSum = everyDayRent * days - 50;
+  } else if (days >= 3) {
+    totalSum = everyDayRent * days - 20;
```

Comment:
- 7, 50, 3, 20 are magic values
<img width="693" alt="image" src="https://github.com/user-attachments/assets/2bdc6d26-2b7d-4028-b017-205ec4beab91">
- Prefer if with return over if else to simplify later conditions.
![Uploading image.png…]()